addons/mail/static/src/new/web/composer_patch.js

```diff
@@ -1,167 +0,0 @@
-                            placeholder: "Loading",
-                            optionTemplate: "mail.Composer.suggestionPartner",
-                            options: mainOrExtraSuggestions.suggestions.map((suggestion) => {
-                                return {
-                                    label: suggestion.name,
-                                    partner: suggestion,
-                                    classList:
-                                        "o-composer-suggestion o-composer-suggestion-partner",
-                                };
-                            }),
-                        };
-                    case "Thread":
-                        return {
-                            placeholder: "Loading",
-                            optionTemplate: "mail.Composer.suggestionThread",
-                            options: mainOrExtraSuggestions.suggestions.map((suggestion) => {
-                                return {
-                                    label: suggestion.displayName,
-                                    thread: suggestion,
-                                    classList: "o-composer-suggestion o-composer-suggestion-thread",
-                                };
-                            }),
-                        };
-                    case "ChannelCommand":
-                        return {
-                            placeholder: "Loading",
-                            optionTemplate: "mail.Composer.suggestionChannelCommand",
-                            options: mainOrExtraSuggestions.suggestions.map((suggestion) => {
-                                return {
-                                    label: suggestion.name,
-                                    help: suggestion.help,
-                                    classList:
-                                        "o-composer-suggestion o-composer-suggestion-channel-command",
-                                };
-                            }),
-                        };
-                    case "CannedResponse":
-                        return {
-                            placeholder: "Loading",
-                            optionTemplate: "mail.Composer.suggestionCannedResponse",
-                            options: mainOrExtraSuggestions.suggestions.map((suggestion) => {
-                                return {
-                                    name: suggestion.name,
-                                    label: suggestion.substitution,
-                                    classList:
-                                        "o-composer-suggestion o-composer-suggestion-canned-response",
-                                };
-                            }),
-                        };
-                    default:
-                        return {
-                            options: [],
-                        };
-                }
-            }),
-        };
-    },
-
-    async sendMessage() {
```

Comment:
Were these intended to be deleted?

Reply:
it was an override of the original composer, I revert the patch since the suggestion is used on the public page as well.